setup.cfg

```diff
@@ -63,7 +63,7 @@ tests_require =
     nbformat
     seaborn
     lightgbm
-    dowhy
+    "dowhy @ git+https://github.com/moprescu/dowhy.git@moprescu/update-econml-args"
```

Comment:
See https://github.com/microsoft/EconML/blob/aa8a6f6aa84bfb2dd73d2851bfcc7cf35d0141d4/setup.cfg#L52 for the correct syntax